kubeflow/kpt-set.sh

```diff
@@ -13,4 +14,4 @@ kpt cfg set -R .  gcloud.compute.zone ${LOCATION}
 kpt cfg set -R .  location ${LOCATION}
 kpt cfg set -R .  log-firewalls false
 
-kpt cfg set -R .  email <YOUR_EMAIL_ADDRESS>
\ No newline at end of file
+kpt cfg set -R .  email <YOUR_EMAIL_ADDRESS>
```

Comment:
nit: include ADMIN_EMAIL var in env.sh? then this can be consistent

Reply:
This is a very good point! I will make the corresponding changes.